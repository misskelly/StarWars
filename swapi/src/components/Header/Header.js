import React from 'react'

const Header = () => {
  return (
    <header className='header'>
      <nav className='main-menu-nav'></nav>
      <h1 className='main-heading'>A <span className='sw-heading'>Star Wars</span>Compendium</h1>
      <nav className='category-nav'></nav>
    </header>
  )

}

export default Header;