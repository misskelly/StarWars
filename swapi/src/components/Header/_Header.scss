.header {
  grid-area: header;
  display: grid;
  grid-template-columns: 2fr 2fr 2fr;
}
ul,
li {
  list-style-type: none;
  margin: 0;
  padding: 0;
}

.nav-left,
.nav-right {
  padding: 10px;
  display: flex;
  justify-content: space-between;
  align-items: center;
}


.nav-left-ul,
.nav-right-ul {
  width: 100%;
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  text-align: center;
}

.main-heading {
  text-align: center;
  font-family: $jedi;
  text-transform: uppercase;
  color: $yellow;
  text-shadow: $outline;
  font-size: 1rem;
}

.sw-heading {
  display: block;
  font-size: 3rem;
  line-height: 1;
}