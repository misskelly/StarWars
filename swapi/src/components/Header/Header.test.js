import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';



describe('Header', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<Header />);
  });
  
  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
  
});