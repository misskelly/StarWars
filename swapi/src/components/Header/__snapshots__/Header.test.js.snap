// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches the snapshot 1`] = `
<header
  className="header"
>
  <nav
    className="nav-left"
  >
    <ul
      className="nav-left-ul"
    >
      <li
        className="main-menu-li"
      >
        <a

          className="main-menu-link"
          href="#"
        >
          Home
        </a>
      </li>
      <li
        className="main-menu-li"
      >
        <a

          className="main-menu-link"

          href="www.instagram.com"
        >
          Learn More
        </a>
      </li>
      <li
        className="main-menu-li"
      >
        <a

          className="main-menu-link"

          href="www.instagram.com"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
  <h1
    className="main-heading"
  >
    A 
    <span
      className="sw-heading"
    >
      StarWars
    </span>
    Compendium
  </h1>
  <nav
    className="nav-right"
  >
    <ul
      className="nav-right-ul"
    >
      <li
        className="collections-menu-li"
      >
        <a

          className="collections-menu-link"

          href="www.instagram.com"
        >
          Planets
        </a>
      </li>
      <li
        className="collections-menu-li"
      >
        <a
          className="collections-menu-link"
          href="www.instagram.com"
        >
          People
        </a>
      </li>
      <li
        className="collections-menu-li"
      >
        <a
          className="collections-menu-link"
          href="www.instagram.com"
        >
          Vehicles
        </a>
      </li>
      <li
        className="collections-menu-li"
      >
        <a
          className="collections-menu-link"
          href="www.instagram.com"
        >
          Favorites
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
